testcases/kernel/controllers/libcontrollers/libcontrollers.c

```diff
@@ -34,6 +34,34 @@
 /******************************************************************************/
 
 #include "libcontrollers.h"
+#include "test.h"
+
+
+volatile int timer_expired;
+int FLAG; //FIXME(mw) - what is the purpose of this variable being global?
+#ifdef PATH_MAX
+char fullpath[PATH_MAX];
+#else
+char fullpath[1024]; /* Guess */
+#endif
+//unsigned int current_shares;
+//unsigned int total_shares;
+//unsigned int *shares_pointer;//??
+int retval;
+struct dirent	*dir_pointer;
+char target[LINE_MAX];
+
+/*
+ * Function: error_function()
+ * Prints error message and returns -1
+ */
+
+static inline void error_function(char *msg1, char *msg2)
+{
+	fprintf(stdout, "ERROR: %s ", msg1);
+	fprintf(stdout, "%s\n", msg2);
+}
+
```

Comment:
Since the `static inline` is an idiom for inline functions defined in headers (to avoid redefinition in a case that the same header is included in both library and program source). It does not make sense to keep the `inline` for a static functions defined in a c source, these would be inlined by the compiler anyway.